spec/page_objects/pages/application.rb

```diff
@@ -34,6 +34,9 @@ def app_pages
         publisher_applying_for_the_job: "Publisher::ApplyingForTheJobPage",
         publisher_how_to_receive_applications: "Publisher::HowToReceiveApplicationsPage",
         publisher_application_link: "Publisher::ApplicationLinkPage",
+        jobseeker_applications: "Jobseeker::JobApplicationsPage",
+        jobseeker_application: "Jobseeker::JobApplicationPage",
+        jobseeker_application_apply: "Jobseeker::JobApplicationApplyPage",
```

Comment:
Huddle with Stephen Dicks
￼
At first I used the page object to test the views  [here was a example](https://github.com/DFE-Digital/teaching-vacancies/blob/7a777c3e9db87d199c3d53ec9bb1e3211242324a/spec/views/jobseekers/job_applications/apply.html.slim_spec.rb) but using `apply_view.load(rendered)`  is deprecated. At first I thought it was just a matter of rendering the layout along with the view but did not solve the deprecation warning. 
I then switch to used Capybara node instead but left the created page objects as it could be useful for future tests